.github/workflows/webpack-jumpstart.yml

```diff
@@ -0,0 +1,80 @@
+name: webpack-jumpstart
+on:
+  pull_request_target:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: none
+    timeout-minutes: 20
+    env:
+      MERGE_SHA: ${{ github.event.pull_request.merge_commit_sha || github.sha }}
+    steps:
+      - name: Build jumpstart tarball
+        run: |
+          id="$(podman container create -u node -w /home/node node /bin/sh -c \
+            'git init cockpit &&
+             cd cockpit &&
+             git fetch --depth 1 https://github.com/cockpit-project/cockpit '${MERGE_SHA}':build &&
+             git checkout build &&
+             pkg/build webpack-jumpstart.tar -j$(nproc)'
+          )"
+
+          podman container start --attach "${id}" >&2
+          podman container cp "${id}":cockpit/webpack-jumpstart.tar .
+          podman container rm -f "${id}"
+
+      - name: Create artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: webpack-jumpstart
+          path: webpack-jumpstart.tar
+          retention-days: 1
+
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: none
+    environmnent: cockpit-dist upload
+    timeout-minutes: 5
+    env:
+      GIT_DIR: git-dir.git
+      MERGE_SHA: ${{ github.event.pull_request.merge_commit_sha || github.sha }}
+    steps:
+      - name: Set up work area
+        run: |
+            set -ux
+
+            git init -b main
+
+            git config user.name "GitHub Workflow"
+            git config user.email "cockpituous@cockpit-project.org"
+
+            git remote add cache "ssh://git@github.com/${GITHUB_REPOSITORY%/*}/cockpit-dist"
+
+      - uses: actions/download-artifact@v2
+        with:
+          name: webpack-jumpstart
+          path: artifact
+
+      - name: Stage git commit
+        run: |
+          set -ux
+          mkdir work-tree
+          tar -C worktree -x --exclude '.git*' dist .sha256sum < artifact/jumpstart.tar
+          git --work-tree worktree add dist .sha256sum
```

Comment:
So this is blocked on #16004.

Reply:
eh.  Let's unblock it and proceed here.  I want to land this, and we can do the sha256sum stuff later.